tests/python/contrib/test_cblas.py

```diff
@@ -158,10 +158,11 @@ def test_quantized_matmul_add():
 
 
 def verify_batch_matmul(
-    batch, m, l, n, lib, transa=False, transb=False, iterative=False, dtype="float32"
+    batch_a, batch_b, m, l, n, lib, transa=False, transb=False, iterative=False, dtype="float32"
 ):
-    ashape = (batch, l, n) if transa else (batch, n, l)
-    bshape = (batch, m, l) if transb else (batch, l, m)
+    batch = batch_a
```

Comment:
If `batch_a == 1` and `batch_b == B`, then `batch` should be `batch_b`?

Reply:
Fixed. Also changed the extern shape calculation.